src/status_im/ui/screens/profile/views.cljs

```diff
@@ -181,6 +184,25 @@
    [info-item-separator]
    [profile-info-phone-item phone]])
 
+(defn save-profile! []
+  (when-let [save-event @(re-frame/subscribe [:my-profile.drawer/save-event])]
+    (re-frame/dispatch [save-event])))
+
+(defn navigate-to-accounts []
+  (save-profile!)
```

Comment:
i think we don't need this anymore

Reply:
Cool, I'll get rid of it in the "real" PR